src/object/class_object.c

```diff
@@ -4017,8 +4018,11 @@ classobj_find_cons_index2_col_type_list (SM_CLASS_CONSTRAINT * cons, CLASS_STATS
       return NULL;		/* give up */
     }
 
-  attr_statsp = stats->attr_stats;
-  for (i = 0; i < stats->n_attrs && !key_type; i++, attr_statsp++)
+  /* Get local stats including invisible indexes. */
+  local_stats = stats_get_statistics (&cons->attributes[0]->class_mop->oid_info.oid, 0);
```

Comment:
It seems `local_stats` will be leaked. 

I want more robust codes for the cases:
`local_stats` might be nil
`cons->attributes[0]->class_mop` might be nil